dist/latin.css

```diff
@@ -2335,12 +2335,12 @@ body {
 }
 
 .gs-o-button--share {
-  background-color: #4a9442;
+  background-color: #0b8a0b;
   color: #fff;
 }
 
 .gs-o-button--share:hover {
-  background-color: #41823a;
+  background-color: #097209;
```

Comment:
It's worth noting that the colour changes in this dist aren't part of the changes for the double alt but brought in as the dist hasn't been built in some time